src/verso.rs

```diff
@@ -162,7 +167,7 @@ impl Verso {
             debug_flags.set(DebugFlags::PROFILER_DBG, opts.debug.webrender_stats);
 
             let render_notifier = Box::new(RenderNotifier::new(compositor_sender.clone()));
-            let clear_color = ColorF::new(1., 1., 1., 0.);
```

Comment:
this change needed?

Reply:
Yes, I need to make sure it's clear with transparent background.